README.md

```diff
@@ -28,94 +28,17 @@ Decidim is a participatory democracy framework written on Ruby on Rails original
 
```

Comment:
I think this is outside of the scope of this PR. Also, I think @josepjaume loves them 😄